core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -23,9 +23,11 @@
 oppia.factory('SimpleEditorManagerService', [
   'StatesToQuestionsService', 'SimpleEditorShimService',
   'QuestionObjectFactory', 'QuestionListObjectFactory', 'OutcomeObjectFactory',
+  'explorationInitStateNameService',
```

Comment:
Place `explorationInitStateNameService` as the first element, as services should be alphabetized.